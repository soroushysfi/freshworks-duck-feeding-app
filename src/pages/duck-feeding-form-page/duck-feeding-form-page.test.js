import { shallow } from 'enzyme'
import DuckFeedingFormPageComponent from './duck-feeding-form-page.component'


it('expect to match DuckFeedingFormPage snapshot', () => {
    expect(shallow(<DuckFeedingFormPageComponent />)).toMatchSnapshot()

})