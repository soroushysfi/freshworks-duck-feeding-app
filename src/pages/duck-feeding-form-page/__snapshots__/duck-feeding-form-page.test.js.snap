// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to match DuckFeedingFormPage snapshot 1`] = `
<div>
  <Connect(DuckFeedingFormComponent) />
</div>
`;
